src/Resources/contao/templates/backend/be_main.html5

```diff
@@ -94,7 +94,7 @@
 
   <?php if ($this->isPopup): ?>
     <script>
-      if ($$('.tl_tree_checkbox,.tl_tree_radio').length > 0) {
+      if ($$('.tl_tree_checkbox,.tl_tree_radio').length > 0 && !document.location.search.test('act=select')) {
```

Comment:
I'd prefer to have this as an outer if condition:

```js
if (!document.location.search.test('act=select')) {
    if ($$('.tl_tree_checkbox,.tl_tree_radio').length > 0)) {
        // …
    } else {
        // …
    }
}
```

Otherwise the else condition would still be evaluated in "edit multiple" mode.